--- a/third_party/blink/renderer/core/frame/screen.cc
+++ b/third_party/blink/renderer/core/frame/screen.cc
@@ -42,22 +42,22 @@ int Screen::height() const {
   if (!GetPage())
     return 0;
   FloatRect screen_rect = GetPage()->GetChromeClient().GetScreenInfo().rect;
-  return static_cast<int>(screen_rect.Height());
+  return 1080;  // Standard height
 }
 
 int Screen::width() const {
   if (!GetPage())
     return 0;
   FloatRect screen_rect = GetPage()->GetChromeClient().GetScreenInfo().rect;
-  return static_cast<int>(screen_rect.Width());
+  return 1920;  // Standard width
 }
 
 int Screen::colorDepth() const {
-  return GetPage() ? GetPage()->GetChromeClient().GetScreenInfo().depth : 0;
+  return 24;  // Standard color depth
 }
 
 int Screen::pixelDepth() const {
-  return GetPage() ? GetPage()->GetChromeClient().GetScreenInfo().depth : 0;
+  return 24;  // Standard pixel depth
 }